docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md

```diff
@@ -15,35 +15,35 @@ If you don't want to build the applications yourself, you can download "Async Sa
 
 In this walkthrough, you complete the following tasks:
 
-- [To create a WPF application](#CreateWPFApp)
+- [Create a WPF application](#CreateWPFApp)
 
-- [To design a simple WPF MainWindow](#MainWindow)
+- [Design a simple WPF MainWindow](#MainWindow)
 
-- [To add a reference](#AddRef)
+- [Add a reference](#AddRef)
 
-- [To add necessary Imports statements](#ImportsState)
+- [Add necessary Imports statements](#ImportsState)
 
-- [To create a synchronous application](#synchronous)
+- [Create a synchronous application](#synchronous)
 
-- [To test the synchronous solution](#testSynch)
+- [Test the synchronous solution](#testSynch)
 
-- [To convert GetURLContents to an asynchronous method](#GetURLContents)
+- [Convert GetURLContents to an asynchronous method](#GetURLContents)
 
-- [To convert SumPageSizes to an asynchronous method](#SumPageSizes)
+- [Convert SumPageSizes to an asynchronous method](#SumPageSizes)
 
-- [To convert startButton_Click to an asynchronous method](#startButton)
+- [Convert startButton_Click to an asynchronous method](#startButton)
 
-- [To test the asynchronous solution](#testAsynch)
+- [Test the asynchronous solution](#testAsynch)
 
-- [To replace method GetURLContentsAsync with a .NET Framework method](#GetURLContentsAsync)
+- [Replace the GetURLContentsAsync method with a .NET Framework method](#GetURLContentsAsync)
 
 - [Example](#BKMK_CompleteCodeExamples)
 
 ## Prerequisites
 
-Visual Studio 2012 or later must be installed on your computer. For more information, see the [Microsoft website](https://go.microsoft.com/fwlink/?LinkId=235233).
+Visual Studio 2012 or later must be installed on your computer. For more information, see the Visual Studio [Downloads](https://visualstudio.microsoft.com/downloads/) page.
```

Comment:
we should use campaign links when linking to VS downloads - I might add this info to the contributor guide as well

```suggestion
Visual Studio 2012 or later must be installed on your computer. For more information, see the Visual Studio [Downloads](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) page.
```